src/internal/tester/cli.go

```diff
@@ -36,7 +36,6 @@ func StubRootCmd(args ...string) *cobra.Command {
 func NewContext() (context.Context, func()) {
 	level := logger.Info
 
-	fmt.Printf("\n-----\nargs %v\n-----\n", os.Args)
```

Comment:
From `src/internal/events/mock/mock.go`
+1 on the check annotations:
```go
type MockBus struct {
```
```bash
exported: type name will be used as mock.MockBus by other packages, and that stutters; consider calling this Bus (revive)
```

Reply:
good old stutters.